src/components/pages/Cart.js

```diff
@@ -30,6 +34,7 @@ export default function Cart () {
                 }
                 <TotalPrice><p>Total: </p> <PriceSpan>R$ {calcTotal().toFixed(2)}</PriceSpan></TotalPrice>
             </CartWrapper>
+            {user.token?<Addresses defaultAddress={defaultAddress} setDefaultAddress={setDefaultAddress} reload={reload} setReload={setReload}/>:""}
```

Comment:
O operador && renderiza apenas se user.token for true/truthy
```suggestion
            {user.token && 
                <Addresses 
                    defaultAddress={defaultAddress} 
                    setDefaultAddress={setDefaultAddress} 
                    reload={reload} 
                    setReload={setReload}
                />
            }
```